blog/ga-patchfix-mars.2.adoc

```diff
@@ -0,0 +1,39 @@
+= Patch fix for Red Hat JBoss Developer Studio 9.0.0.GA and JBoss Tools 4.3.0.Final
```

Comment:
Patch fix is a political loaded word - and the title is a mile long ;)

Suggestion:
Fix for Developer Studio and Tools updatesites for Eclipse Mars.2